airbyte-integrations/singer/exchangeratesapi/source/src/test-integration/java/io/airbyte/integration_tests/sources/SingerExchangeRatesApiSourceDataModelTest.java

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
are the java files in here just lift and shifts?

Reply:
yep duplicated for testing purposes.